xml_inscriptions/transcribed/MA.Bos.MFA.G.03.997.xml

```diff
@@ -0,0 +1,183 @@
+                  <provenance type="transferred" subtype="sold">
+                     <p> <date when="1898">1898</date>: auction of the <persName>M. Tyszkiewicz</persName>Collection, Hotel des Commissaires-Priseurs, 9 rue Drouot, <placeName>Paris</placeName>, <date>June 8-10</date>, lot 133 (trouvée à Thèbes)</p>
+                  </provenance>
+                  <provenance type="observed" subtype="recorded">
+                     <p> by <date when="1897">1897</date>: in <persName>Count Michel Tyszkiewicz</persName>Collection</p>
+                  </provenance>
+                  <provenance type="observed" subtype="recorded">
+                     <p> According to <persName>W. Fröhner</persName>, La Collection Tyszkiewicz (1897), p. 40: found in
+                        <placeName>Thebes</placeName> and sent to <persName>M. Hoffmann</persName> toward the end of <date when="1894">1894</date></p>
+                  </provenance>
+                  
+                  
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CVB"/>
+                  <biblScope unit="item">no. 15</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#LSAG"/>
+                  <biblScope unit="item">94, no. 1</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="https://collections.mfa.org/objects/152660/mantiklos-apollo?ctx=f793a1e4-8834-4625-9d08-7494e81f8367&amp;idx=0"
+                     >MFA web entry</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-18" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.03.997"/>
+      </surface>
+   </facsimile>
+   
+   <text>
+      <body>
+         <div type="edition">
+            <ab><lb n="1"/>ΜΑΝΤΙΚΛΟΣΜΑΝΕΘΕΚΕFΕΚΑΒΟΛΟΙΑΡΓΥΡΟΤΟΧΣΟΙΤΑΣΔΕΚΑΤΑΣΤΥΔΕΦΟΙΒΕΔΙΔΟΙΧΑΡ</ab></div>
```

Comment:
Transcription needs to be interpretive (i.e. include diacritics), or at least have spacing given for the word breaks

Reply:
I fixed